### ISSUE TITLE
Allow deletion of breakpoints with 'delete' key & select multiple breakpoints to delete

### ISSUE THREAD

**Issue by cdacamar** [Microsoft/vscode#45364]
- VSCode Version: 1.21.0
- OS Version: Windows 10 Build 16299.rs3_release.170928-1534

Steps to Reproduce:

1. Try to delete breakpoints with 'delete' key
2. Try to select multiple breakpoints

Does this issue occur when all extensions are disabled?: Yes

**Comment by kieferrm** [comment]
Keyboard support in the breakpoints viewlet is bad. `delete` does not work, I need to press `enter` to move the keyboard focus to other list items. I can't use `space` to check or uncheck checkboxes.

**Comment by tsalinger** [comment]
It's possible now to 
- select breakpoints in the viewlet with space
- delete a single breakpoint with the delete key

It's not possible to delete multiple selected breakpoints with the delete key.
Also, the multi selection works differently than in the explorer:
1. select a breakpoint with the mouse
2. hold shift and down arrow to multi-select
3. click on the first breakpoint again
4. repeat 2.

Notice how the selection skips to one past the last item from step 2.
![breakpointshortcuts](https://user-images.githubusercontent.com/15158320/38040209-430c405e-32af-11e8-896a-88f6cb2c6712.gif)

**Comment by isidorn** [comment]
@tsalinger good catch, however the fix for this item where more aiming to make the view accessible, not multi select aware. For that I am still waiting for feature reqeusts from users to polish it
